Gemfile

```diff
@@ -8,7 +8,7 @@ end
 gem 'rails', '~> 5.1'
 
 gem 'autoprefixer-rails'
-gem 'axlsx'
+gem 'axlsx', '~> 2.1.0.pre'
```

Comment:
When you specify a specific version I would always add a comment with the reason why (e.g. link to bug report), this can save a lot of time when upgrading the app in the future.

Reply:
If I only know that axlsx is failing without and axlsx documentation tells me to do so?